<!---->
<launch>

<!---assemble clouds and aply a voxel grid-->
	<node pkg="sonar_cloud" type="sonar_cloud_assembler" output="screen" name="assemble_sonar">
	<param name="sonar_in" value="/sonar_data" />
	<param name="sonar_cloud" value="/sonar_cloud" />
	<param name="fixed_frame" value="/odom" />
	<param name="sonar_frame" value="/sonar_frame" />
	<param name="apply_voxel" value="true" />
	<param name="x" value="0.001" />
	<param name="y" value="0.001" />
	<param name="z" value="0.001" />
	<param name="frequency" value="1" />
	

	</node>
</launch>
